### PR TITLE
E1.33 cherry pick third time lucky

### PR DESCRIPTION
**Pull request by peternewman**
But maybe not for you, apologies I probably pulled a bit too much into this sorry.

Let me know if it's too much and I'll rethink somehow...

More of #1841

### CODE REVIEW

File: libs/acn/BrokerPDU.cpp
Comment (kripton) [inline libs/acn/BrokerPDU.cpp:45]: I´d add curly braces here

**Review by kripton — APPROVED** [review]
Nice work! See inline comments.
Keep 'em coming :)

File: plugins/uartdmx/README.md
Comment (peternewman) [reply plugins/uartdmx/README.md:15]: I thought that too so initially went and removed them from everything and then found it rendered oddly as it became an inline code block. Maybe if we changed it to a triple backtick it might work possibly?

File: plugins/kinet/README.md
Comment (peternewman) [reply plugins/kinet/README.md:11]: As above.

File: include/ola/e133/E133Enums.h
Comment (peternewman) [reply include/ola/e133/E133Enums.h:100]: Just a simple typo, now fixed.

File: libs/acn/BrokerPDU.cpp
Comment (peternewman) [reply libs/acn/BrokerPDU.cpp:56]: Good call, something I used to ask Simon for and missed in the copy/paste. Done

File: libs/acn/BrokerPDU.cpp
Comment (peternewman) [reply libs/acn/BrokerPDU.cpp:45]: Done again.

File: libs/acn/LLRPProbeReplyInflator.h
Comment (peternewman) [reply libs/acn/LLRPProbeReplyInflator.h:75]: Noted, I'll push that all to there for one big bang. I'd like to get 0.11.0 out with this in before ripping through the code too much.

File: libs/acn/LLRPProbeRequestInflator.h
Comment (peternewman) [reply libs/acn/LLRPProbeRequestInflator.h:67]: As above

File: libs/acn/RDMInflator.h
Comment (peternewman) [reply libs/acn/RDMInflator.h:73]: As above

File: libs/acn/LLRPProbeRequestInflator.cpp
Comment (peternewman) [reply libs/acn/LLRPProbeRequestInflator.cpp:113]: No it's actually effectively a TODO, we shouldn't be responding unless some filter flags match and we're not in the UIDSet of known UIDs. I've turned it into a TODO to probably add them to the callback, and will then need to action them in code that uses it, but that's a later challenge, the E1.33 broker stuff is probably a bit more time sensitive currently.

File: libs/acn/LLRPProbeRequestInflator.h
Comment (peternewman) [reply libs/acn/LLRPProbeRequestInflator.h:43]: Again.

File: libs/acn/LLRPProbeRequestInflator.cpp
Comment (peternewman) [reply libs/acn/LLRPProbeRequestInflator.cpp:113]: Also now added to #1612 to make sure it doesn't get missed.

File: include/ola/e133/E133Enums.h
Comment (peternewman) [reply include/ola/e133/E133Enums.h:97]: Ask @nomis52 from 11 years ago! :laughing: 
https://github.com/OpenLightingProject/ola/blame/f0b01b359b4bfc3e24eb0a7a2aa96028638128ef/include/ola/e133/E133Enums.h#L95

I just copied it for the version, but I guess we could theoretically support a few versions at the same time so maybe they should be an enum at least?

Comment (kripton) [reply include/ola/e133/E133Enums.h:97]: Sure, if multiple values make sense, keep it as an enum. How about this MAX_STRING_SIZE?

File: plugins/uartdmx/README.md
Comment (kripton) [reply plugins/uartdmx/README.md:15]: Mh, I also cannot figure why that would render strange. If adding white space fixes it, just keep that

File: libs/acn/LLRPProbeReplyInflator.h
Comment (kripton) [reply libs/acn/LLRPProbeReplyInflator.h:75]: Sure, no problem, one step after the other

File: libs/acn/LLRPProbeRequestInflator.cpp
Comment (kripton) [reply libs/acn/LLRPProbeRequestInflator.cpp]: Sure, no problem, one step after the other

File: libs/acn/LLRPProbeRequestInflator.h
Comment (kripton) [reply libs/acn/LLRPProbeRequestInflator.h]: Sure, no problem, one step after the other

File: libs/acn/LLRPProbeRequestInflator.h
Comment (kripton) [reply libs/acn/LLRPProbeRequestInflator.h:67]: Sure, no problem, one step after the other

File: libs/acn/RDMInflator.h
Comment (kripton) [reply libs/acn/RDMInflator.h:73]: Sure, no problem, one step after the other

**Comment by kripton** [comment]
> I've pushed. Do you want to re-review or are you happy if I just merge @kripton ?

You can merge :+1: . If I approve on the first review, it's just minor stuff that I trust people to fix (or at least triage) before they merge. If it something I want to check again, I finish the review with "Comment only, no approval" :)

File: include/ola/e133/E133Enums.h
Comment (peternewman) [reply include/ola/e133/E133Enums.h:97]: I'm not entirely sure they do necessarily. I've kicked it down the road a bit and added a todo in #1841 anyway.

**Comment by peternewman** [comment]
> You can merge 👍 . If I approve on the first review, it's just minor stuff that I trust people to fix (or at least triage) before they merge.

Cool, I think I'll park a chunk of the more complex ones or those that require a bit more thought/clean-up into the main issue to try and avoid conflicts and tidying up multiple times as different bits are merged. I think we're only around 1/3 to 1/4 of the way through so far unfortunately. 

> If it something I want to check again, I finish the review with "Comment only, no approval" :)

Or maybe you could use one of the specific options GitHub offers, then it's not actually possible to merge until you get an approval? :)

> Comment
> Submit general feedback without explicit approval.
> Approve
> Submit feedback and approve merging these changes.
> Request changes
> Submit feedback that must be addressed before merging.

**Comment by peternewman** [comment]
Force merging as the CI error is already covered in #1948 .

File: libs/acn/LLRPProbeRequestInflator.cpp
Comment (peternewman) [reply libs/acn/LLRPProbeRequestInflator.cpp]: Turns out future me (or I guess past me from this perspective) had already fixed it already in a later commit which will get in eventually! https://github.com/OpenLightingProject/ola/commit/f2343ce127c43a395ffb5dec1457bc3165b02b6a

**Pull request merged**
